README.md

```diff
@@ -1,5 +1,18 @@
 # GMT 中文社区维护指南
 
+本项目用于记录 gmt-china 的相关信息。
+
+## 基本信息
+
+- 网站名：GMT 中文社区
+- 主页：https://gmt-china.org
+- GitHub 组织：https://github.com/gmt-china
+
+## 域名信息
+
+- 所有者：[@seisman][seisman]
```

Comment:
```suggestion
- 所有者：[@seisman](https://github.com/seisman)
```

Reply:
It works because there is a definition for this link at the end of the file.